src/test/java/seedu/address/logic/commands/SortCommandTest.java

```diff
@@ -0,0 +1,27 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code SortCommand}.
+ */
+public class SortCommandTest {
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
```

Comment:
i think for the SortCommandTest, you can create a different method like a getUnsortedAddressBook() to have an unsorted version of the getTypicalAddressBook() as currently now the getTypicalAddressBook() already gives a sorted list